### ISSUE TITLE
CP1250 encoding problem from odbc data

### ISSUE THREAD

**Issue by ctahk0** [markdirish/node-odbc#40]
Hello,

I have so many problems with non-unicode characters from database. 
The database is IBM Informix, and DB_LOCALE=cs_cz.cp1250 is set in odbc.ini.
I can get correct result when run from ISQL (smal SQL tool installed with Informix SDK), the result is like bellow:
![image](https://user-images.githubusercontent.com/7126829/65453206-03d55b00-de43-11e9-9727-7e40e1be0d9b.png)

Putty terminal is set to CP1250 
![image](https://user-images.githubusercontent.com/7126829/65453422-73e3e100-de43-11e9-9057-ed4388e334e1.png)
So, I guess ODBC works correctly.

But, from node, with node-odbc this simple code gives totally wrong results:

`const odbc = require('odbc');

async function connectToDatabase() {
	const connectionConfig = {
		connectionString: 'DSN=cistoca;encoding=cp1250',
		connectionTimeout: 10,
		loginTimeout: 10,
	}
	const connection2 = await odbc.connect(connectionConfig);

	connection2.query("SELECT naz_par FROM pos_par LIMIT 10", function (err, data) {
		if (err) {
			console.log('Error from connection!', err);
			return;
		}
		for (var i = 0; i < data.length; i++) {
			console.log(data[i].naz_par);
		}
	});
	// connection2 is now an open Connection
}
connectToDatabase();`

Here is the result:
![image](https://user-images.githubusercontent.com/7126829/65453959-7abf2380-de44-11e9-95ba-3933aab5be8a.png)

When I switch Terminal to UTF-8 I got this:
![image](https://user-images.githubusercontent.com/7126829/65454097-c1148280-de44-11e9-84f6-be32dd14d1f0.png)

And when run again program, here is the result:
![image](https://user-images.githubusercontent.com/7126829/65454163-d689ac80-de44-11e9-91e9-0015c825fb72.png)

I'm desperate and don't know how to resolve this.
I tried literally 70+% of the Stack Overflow before I ask here for your help, but without success.

Thank you in advance!
Stanko

**Comment by worksofliam** [comment]
Hi @ctahk0 

Are you able to provide some SQL statements (create+inserts) for us to recreate the table with the correct columns and some of the data you have? This will make it easier for us to debug. Thanks!

**Comment by ctahk0** [comment]
Unfortunately I can't insert or edit data, I have only read access. The db is Informix and works on windows env without any problems.

**Comment by ctahk0** [comment]
What I can is ask admin to make copy of db and give you ssh access?

**Comment by ctahk0** [comment]
Hi,
I want to inform you that I was able to get the correct characters.
The problem is not in your code at all (fortunately :) )
After setting up the correct client locale in odbc.ini I got the correct results.
In my case this settings works:
![image](https://user-images.githubusercontent.com/7126829/65528076-08a61780-def4-11e9-8df0-d711605e1e9a.png)

Instead of this (wrong results):
![image](https://user-images.githubusercontent.com/7126829/65528157-2b383080-def4-11e9-8e0d-1d0e09dfe6e1.png)

Thanks again.
